python_modules/libraries/dagster-k8s/dagster_k8s/client.py

```diff
@@ -222,6 +224,18 @@ def k8s_api_retry_creation_mutation(
                     k8s_api_exception=e,
                     original_exc_info=sys.exc_info(),
                 ) from e
+        except urllib3.exceptions.HTTPError as e:
```

Comment:
do we know anything about the specific type of HTTPError or we actually want to cover all of them?

Reply:
Protocol error is such a strange exception to raise for RemoteDisconnected. I think I have a slight preference for being more granular here though the downside of retrying a non-retriable error is probably low given the max retries escape hatch...